web_s2i/templates/layout.html

```diff
@@ -35,6 +35,24 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
 
+    <!-- Default Statcounter code for Devopsloft  -->
+    <script type="text/javascript">
+    var sc_project=11982438;
+    var sc_invisible=0;
+    var sc_security="03b2cd4a";
+    var scJsHost = "https://";
```

Comment:
does it mean the site is https://www.deveoploft.io? we haven't implemented it yet.

Reply:
yes

Reply:
no, sorry - I wrote done http:// ..

Reply:
I am not following

Reply:
https is to the site statcounter.com
I implemented the following URL:  http://www.devopsloft.io/

Reply:
I defined the following URL as open for the public
do you want me to add this to the readme?

Reply:
I don't think it is required in the README. But, that is my 2 cents. Maybe others think differently.

Reply:
I have just googled and found recommendation for having different **statcounter** project for each environment.
Any idea what is the recommended way to pass parameters to the app per environment ?

Reply:
You want the app to "know" in which environment it is running?

Reply:
Yes, 
So we will not mix statistics among environment.
Currently any web access is counted, even dev

Reply:
We know the IP address of the PROD environment - 52.10.238.106. Can you use it?

Reply:
I assume that we can. the application.py can know by IP that it is PROD.
but I prefer to have statistics also for stage and dev - separately 
This way we will see how much people used our code :)

Reply:
Ohh, do you want to collect stats on how many DEV/STAGE/CI environment spinned. Let me think a bit.

Reply:
I think, Vagrant can inject an environment variable to the VM and from the VM to the running app container.